src/main/java/frc/team2412/robot/util/auto/AutoLogic.java

```diff
@@ -474,7 +479,7 @@ public static Command feederIn() {
 		return (INTAKE_ENABLED && LAUNCHER_ENABLED
 						? Commands.waitUntil(isReadyToLaunch())
 								.andThen(Commands.waitSeconds(FEEDER_DELAY))
-								.andThen(new FeederInCommand(s.intakeSubsystem))
+								.andThen(new FeederInCommand(s.intakeSubsystem).until(untilNoNote())
 								.andThen(Commands.waitSeconds(0.1))
```

Comment:
Would be nice to name this "feed until note launched", to differentiate from "feed until note detected". I assumed `feederIn` meant the latter. And bonus if you add a `.withName("myFancyName")` so the command has a meaningful name in the logs.